stories/calendar-input.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import {storiesOf, addDecorator} from '@storybook/react'
+import {Calendar} from '@smashing/calendar-input'
+import {withA11y} from '@storybook/addon-a11y'
+import {SmashingThemeProvider} from '@smashing/theme'
+
+addDecorator(withA11y)
+
+storiesOf('Core|Calendar', module)
+  .addDecorator(story => (
+    <SmashingThemeProvider
+      theme={{
+        defaults: {
+          calendar: {}
+        }
+      }}
+    >
+      {story()}
+    </SmashingThemeProvider>
+  ))
+  .add('appearance:default', () => {
+    const [date, setDate] = React.useState(new Date())
```

Comment:
You shouldn't use hooks directly in stories - seemingly it >might< work, but actually can cause fatal exceptions upon mounting/unmounting, as well as in displaying storybook docs. It's better to wrap story contents in additional component layer.